control-plane/rest/openapi-specs/v0_api_spec.yaml

```diff
@@ -2226,6 +2228,8 @@ components:
           $ref: '#/components/schemas/NodeId'
         status:
           $ref: '#/components/schemas/NodeStatus'
+        node_nqn:
+          $ref: '#/components/schemas/HostNqn'
```

Comment:
Same comment as above.

Reply:
Hey @avishnu this has been merged now, but just for clarification, hostnqn is the actual nvme terminology.

Reply:
Ok, I was thinking about the naming consistency. There are "schemas/NodeId", "schemas/NodeStatus" and then there is "schemas/HostNqn", hence the comment.

Thanks !!